framework/include/Player.h

```diff
@@ -0,0 +1,38 @@
+#ifndef _PLAYER_H
+#define _PLAYER_H
+
+#include <vector>
+#include <string>
+
+#include "GameBoard.h"
+
+namespace Player {
+
+// Player type
+// Used to store which type of player (AI)
+typedef uint8_t playertype_t;
```

Comment:
would it be better to call this player_t rather than playertype_t. kinda sounds like playertype_type

Reply:
True, I don't know what I was thinking. I renamed it.